packages/node-integration-tests/suites/public-api/startTransaction/manual-tracing-namespace-import/test.ts

```diff
@@ -1,4 +1,4 @@
-import { assertSentryTransaction, getEnvelopeRequest, runServer } from '../../../utils';
+import { assertSentryTransaction, getEnvelopeRequest, runServer } from '../../../../utils';
 
 test.skip('should send a manually started transaction when @sentry/tracing is imported using namespace import.', async () => {
```

Comment:
Should this test still be skipped?

Reply:
Kept that in case we decide to support this in future, can just delete though.

Reply:
Let’s just delete for now